frontend/src/components/JsonViewer/utils.ts

```diff
@@ -0,0 +1,71 @@
+import {
+	AndOrExpression,
+	SearchExpression,
+} from '@/components/Search/Parser/listener'
+import { ReservedLogKey } from '@/graph/generated/schemas'
+
+const bodyKey = ReservedLogKey['Message']
+
+export const findMatchingAttributes = (
+	queryParts: Array<SearchExpression | AndOrExpression>,
+	logAttributes: object | string,
```

Comment:
Nit: should we rename this to `attributes`?

Reply:
Good call - will update before merging!